src/TextField/TextField.js

```diff
@@ -73,6 +76,15 @@ export default class TextField extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
+    /**
+     * Whether the label should be displayed in an error state
+     */
+    error: PropTypes.bool,
+    /**
+     * Whether this label should indicate that the input
+     * is required.
+     */
+    required: PropTypes.bool,
```

Comment:
Alright, so it's just a helper. I'm not sure. I mean. Why doing it for this property and not the other one? I think that we would need a more generic approach if we start doing so.
`shrink` case is a bit different as it's needed for the state.